### PR TITLE
Fixes for the crash issue when the time is uninitialized

### PR DESCRIPTION
**Pull request by iWumboUWumbo2**
I don't know if this is the best way to resolve the problem because as @Zangetsu38 mentioned, the GUI is very complicated but this at least mitigates the crashing.

### CODE REVIEW

File: vita3k/gui/src/gui.cpp [hunk 1]
```diff
@@ -435,6 +436,7 @@ void init_home(GuiState &gui, HostState &host) {
     }
 
     get_time_apps(gui, host);
+    init_last_time_apps(gui, host);
```
Comment (Zangetsu38) [inline vita3k/gui/src/gui.cpp:439]: Some is fine but here is wrong, like i explain is complicated

**Comment by sunho** [comment]
Is the edition to app_selector.cpp still needed after the init fix?

**Comment by sunho** [comment]
by init fix I'm referring to last_time=0

**Comment by Zangetsu38** [comment]
> by init fix I'm referring to last_time=0

Yes both no request, in normal mode value is obligatory 0 if not

**Comment by Zangetsu38** [comment]
Code here no fix issue att all is more Skipped,  you obligiyary need run init last time after init user,i have this and founded how fix it quickely, thx for your works 👌

**Pull request closed without merging**